### ISSUE TITLE
./large-card.css missing

### ISSUE THREAD

**Issue by ggascoigne** [Tripwire/octagon#123]
After upgrading from 7.2.0 I get:
```
./node_modules/react-octagon/lib/components/LargeCard/LargeCard.js
Module not found: Can't resolve './large-card.css' in 'node_modules/react-octagon/lib/components/LargeCard'
```

**Comment by cdaringe** [comment]
did you check to see if indeed the file is a sibling?  if so, sounds like one of your loaders is going haywire.  im skeptical that this is a concern of this lib.  the css file existed and worked before you upgraded, no?

**Comment by ggascoigne** [comment]
Well upgrading caused the error and reverting fixed it. I didn't have time
to work out where the exact problem was.  I'll dig.

**Comment by cdaringe** [comment]
closed by #125
